delphi/MarkdownUITestForm.dfm

```diff
@@ -11,7 +11,8 @@ object MarkDownUITest: TMarkDownUITest
   Font.Name = 'Tahoma'
   Font.Style = []
   OldCreateOrder = False
-  PixelsPerInch = 96
+  Scaled = False

```

Comment:
IMHO form scaling should not be on, because it is scaled wrong on different DPIs

Reply:
I run on a 4K screen. It looks truly bizarre when not scaled. I'm going to turn scaling back on for now, but surely there's a way to do this that works?